eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -37,7 +39,7 @@ jobs:
     ${{ if eq(parameters.compilerName, 'gcc') }}:
       name: ${{ format('coreclr_{0}_product_build_{1}{1}_{3}_{4}', parameters.compilerName, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
```

Comment:
Is it a bug that this says `{1}{1}` instead of `{1}{2}`? Looks like argument 2 is never used.

Reply:
Yes, I believe this is clearly a bug. In fact, I'm wondering why name needs to be conditional at all considering we're not encoding the compiler specifier in it. That would naturally only be a concern if we had combined pipelines doing both GCC and CLANG compilation.

Reply:
Yeah, that is definitely a bug, it is duplicating argument 1 in the name.

Reply:
It's just weird that the compilerName gets "overlaid" with the runtimeVariant in the first variable name component, I guess we should just have both in the name but feel free to ignore this for now if you don't feel comfortable expanding your change in such a non-trivial way.